traffic_ops/traffic_ops_golang/trafficvault/backends/postgres/encrypt.go

```diff
@@ -32,7 +32,7 @@ import (
 	"github.com/apache/trafficcontrol/traffic_ops/traffic_ops_golang/trafficvault/backends/postgres/hashicorpvault"
 )
 
-func aesEncrypt(bytesToEncrypt []byte, aesKey []byte) (string, error) {
+func AesEncrypt(bytesToEncrypt []byte, aesKey []byte) (string, error) {
```

Comment:
exported symbols should have GoDocs (e.g. is that error safe to show to a client, or only for logging server-side?) and nit but as an initialism AES should probably be capitalized